player/lua/stats.lua

```diff
@@ -430,6 +430,8 @@ local function add_file(s)
         append_property(s, "media-title", {prefix="Title:"})
     end
 
+    append_property(s, "file-format", {prefix="Format/protocol:"})
```

Comment:
for consistency this should say `"Format/Protocol:"`

Reply:
Done.